_typos.toml

```diff
@@ -0,0 +1,3 @@
+[default.extend-words]
+froms = "froms"
+fo = "fo"
```

Comment:
Should we just change those to some valid word, to avoid false negatives?

Reply:
Im on it.

Reply:
Thanks tons

Reply:
Np. Done.